Earthworks-01/UT-Earthworks-01.ifc

```diff
@@ -0,0 +1,45 @@
448),(1138,1448,1566),(1451,1445,1695),(1172,1448,1138),(1566,979,1172),(1091,980,981),(1566,1172,1138),(1844,1377,768),(1695,1445,1128),(1225,966,1255),(1566,1838,979),(1644,492,1204),(1438,1188,1694),(1225,967,1315),(1148,806,1348),(1225,1686,1062),(1445,1062,1444),(1254,1444,1664),(1876,1913,978),(972,1367,1735),(1625,532,1022),(1913,976,1522),(1426,611,1154),(1664,1444,1293),(1855,533,1490),(1393,1208,1210),(954,1118,1775),(1877,1175,560),(1137,1242,965),(1238,1622,525),(1570,1095,1139),(1534,1398,750),(1118,953,1476),(477,1208,1127),(1304,571,1715),(1137,961,962),(796,1148,1538),(1710,1439,1221),(1991,1483,1481),(1800,1175,1877),(560,1280,1877),(1225,968,1451),(960,959,1521),(1292,1957,569),(959,1116,1521),(1116,959,1229),(1359,1141,1280),(1060,958,959),(1494,1443,958),(1766,1743,1163),(1443,1494,1190),(957,1190,1494),(1770,1392,1833),(779,1668,784),(1281,1117,957),(1117,1281,1442),(1163,1404,1766),(1442,1766,1404),(1110,1165,1129),(1404,1163,1441),(1417,1096,1661),(840,1126,851),(1441,1743,1594),(1379,525,1438),(1664,1520,1254),(1040,1770,462),(1594,955,1807),(1923,1807,955),(1095,1570,1668),(851,819,840),(1775,1935,1923),(1479,1149,1162),(1137,962,1679),(953,1118,954),(1569,492,494),(548,1547,1152),(1148,796,806),(1299,1476,953),(1457,817,819),(1221,1476,1299),(1205,1041,1273),(960,1521,1619),(1394,1201,1857),(572,1419,589),(1345,1754,1304),(1351,1121,1635),(1022,1409,1368),(1422,481,1395),(1993,1329,415),(1152,1547,1800),(1489,607,1350),(1150,1259,806),(1120,1351,1635),(1785,1370,224),(1066,868,869),(1329,1993,423),(1066,1121,928),(1121,1066,1437),(1602,1290,445),(1730,869,1122),(869,1730,1066),(868,1122,869),(1134,1555,1135),(1159,589,1419),(851,1436,868),(1126,840,1144),(955,1775,1923),(606,1350,710),(819,1144,840),(1144,819,1435),(1435,817,1130),(611,708,1508),(1824,1418,1110),(1732,768,1140),(1105,1130,817),(1130,1105,1131),(1508,607,611),(1994,635,708),(546,1168,547),(1139,779,1431),(1538,1131,1105),(1131,1538,1434),(1434,1148,1133),(1957,1292,597),(1291,445,446),(510,509,1209),(1133,1348,1259),(1227,928,1351),(1294,1406,1093),(755,1140,768),(1538,1052,796),(1654,1797,298),(1323,1364,1859),(1957,1164,569),(1052,1528,796),(1528,1586,1134),(1586,1528,1052),(1668,1432,784),(1416,553,1166),(1378,321,320),(784,1135,1555),(1135,784,1432),(560,597,606),(532,546,547),(1432,1668,1570),(817,1538,1105),(1565,1170,481),(779,1139,1095),(491,489,1038),(1431,1658,1732),(1658,1431,779),(792,1490,546),(1074,1732,1658),(560,606,1280),(768,1732,1074),(1123,1188,525),(806,796,1150),(510,1191,1622),(742,1140,755),(1003,1371,239),(606,1423,1489),(1157,1112,1420),(1586,784,1555),(1280,1141,1877),(477,928,1227),(321,1378,1759),(611,1519,1994),(1668,779,1095),(1681,1350,1613),(710,1681,1359),(606,710,1280),(1350,1681,710),(1407,514,1189),(1832,1154,607),(357,1384,362),(1154,1832,1155),(1384,357,1787),(708,1145,1508),(597,1156,606),(1475,1731,554),(1602,454,1770),(1155,1489,1423),(1162,1145,708),(1145,1162,1149),(708,635,1479),(1112,1157,593),(635,1149,1479),
```

Comment:
I'd rather see a real, volumetric "body" shape here, like a tessellated solid.

Reply:
@pjanck, I agree with you, but in our example Earthworks fill is represented only with the upper surface of the embankment. So I don’t know what I can use instead of “Tessellation”. Maybe "Brep" or what do you suggest?

Reply:
Can you somehow calculate the whole boundary? That is, produce a true solid body?

Reply:
I have now created a volumetric body for the embankment. BricsCAD allows you to create a Mesh or 3D Solid between two TIN Surfaces. Since it can be no longer represented with ifcTriangulatedIrregularNetwork (vertical surfaces) I used IfcTriangulatedFaceSet.

Reply:
looks god and resolved!